plugins/database/src/low_level_odbc.cpp

```diff
@@ -456,7 +453,7 @@ _cllExecSqlNoResult(
     icatSessionStruct* icss,
     const char*        sql,
     int                option ) {
-    rodsLog( LOG_DEBUG10, "%s", sql );
+    log_sql::debug("{}: SQL = [{}]", __func__, sql);
```

Comment:
Please make all appearances of `SQL` consistent. I've seen it in uppercase and lowercase form now. Using the same case is ideal.